proxy/nginx/lualib/router.lua

```diff
@@ -18,9 +18,10 @@ local redis_port = os.getenv("CERYX_REDIS_PORT")
 if not redis_port then redis_port = 6379 end
 local res, err = red:connect(redis_host, redis_port)
 
--- Return if could not connect to Redis
+-- Exit if could not connect to Redis
 if not res then
-    return
+    ngx.log(ngx.ERR, "failed to connect to redis: " .. err)
+    ngx.exit(ngx.HTTP_INTERNAL_SERVER_ERROR)
```

Comment:
HTTP 500 is really not informative enough. I wasn't sure about what response to return to the client, maybe a custom error page (or json response)?